### ISSUE TITLE
An error occurred while processing a template (src is not defined)

### ISSUE THREAD

**Issue by omundy** [bylexus/grunt-md2html#23]
I receive this error when trying to access `src` from a template either as `<%= title(src) %>` or `<%= src %>`

```bash
Warning: An error occurred while processing a template (src is not defined). Use --force to continue.
```

Could it be this line?
https://github.com/bylexus/grunt-md2html/blob/1fae3a8351cd0c87bd92c702efbe9a2cdba821f4/tasks/md2html.js#L107

**Comment by bylexus** [comment]
Hi,

Thanks for your input. Problem here is that `src` would only contain a meaningful value if the original .md file maps 1:1 to an output HTML file: As soon as multiple .md files are concatenated into one output HTML file, "src" will point to multiple files (at the moment, it would just point to the LAST input file, which is surely wrong).

So `src` need to be either a single file path, OR an array of original files. Would that be a feasible solution?

At the moment, the documentation and the implementation clearly differ for that template var.

**Comment by omundy** [comment]
Thanks for looking into this. It seems the feature to combine multiple files is less about creating an array of content than it is to, like much of the web, include multiple partials within a view? That said, my use case is that I want to get the md file name to print the title in html. So as long as the file name is accessible (as the first) in an array or through another means it will work for me. 🙂

**Comment by bylexus** [comment]
Please note that the missing `src` property is only missing in the layout file: the single content files get the `src` property as expected: Only if your layout file contains the `<%= src %>` variable (which you obviously need if you want to use it as title), then this error occurs

I'm working on a fix for that, and let you know.

**Comment by bylexus** [comment]
Fixed in 0.6.1, let me know if you have any other input.

**Comment by omundy** [comment]
Works great. Thanks so much for the quick attention.
